components/Slider/src/MDCSlider.m

```diff
@@ -385,6 +385,11 @@ - (BOOL)isTracking {
 
 #pragma mark - UIView methods
 
+- (void)setExclusiveTouch:(BOOL)exclusiveTouch {
+  [super setExclusiveTouch:exclusiveTouch];
+  _thumbTrack.exclusiveTouch = exclusiveTouch;
```

Comment:
Is it the thumb track that ultimately responds to touches?

Reply:
Yes, the thumb track is what actually handles all the touch events. I've tested this change in my app and it works as expected.